app.py

```
@@ -112,17 +127,23 @@ def dashboard_dump():
         stop = data.get("stop")
 
-        if bool(start) == bool(stop): return "Only start or stop can be set"
+        if bool(start) == bool(stop): return Response("Only start XOR stop can be set", status=400)
 
         if start:
-            stop_event.clear()
-            dashboard_dump_thread.start()
-            return "dashboard-dump poll started"
+            if dashboard_dump_thread.is_alive(): # Protective case
+                return Response("Polling was already running", status=200)
+            else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)